SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -14,6 +14,8 @@
 #include "Misc/Paths.h"
 #include "Modules/ModuleManager.h"
 
+constexpr int SECONDS_TO_MILLIS = 1000;
```

Comment:
I like the sentiment, but I think standard manipulations like this can be done in place.